.circleci/config.yml

```diff
@@ -18,12 +18,13 @@ jobs:
       - run:
           name: Install requirements
           command: |
-            pip install -r requirements.txt
-            pip install -r requirements-viz.txt
-            pip install -r requirements-dev.txt
+            rm `which poetry`
+            curl -sSL https://install.python-poetry.org | python -
```

Comment:
suggestion: maybe let's use the base image `cimg/python:3.8`, which comes with poetry pre-installed?
See [here](https://github.com/up42/sdk-tests/blob/64736c0c9e9beb5bc3ca1d26e345e6a6f6d7e5c9/.circleci/config.yml#L5) for reference

Reply:
might also be a good opportunity to start using 3.11 as base image. But let's tackle that in another ticket as this might get out of scope (if some tests start failing)

Reply:
Yes, problem that I mentioned before. For the python 3.11 as base image, what we can do actually is test the SDK for different versions, this should work on all versions that we support.